src/tools/run-make-support/src/rustdoc.rs

```diff
@@ -137,6 +143,13 @@ impl Rustdoc {
         self
     }
 
+    /// Add a directory to the library search path.
```

Comment:
Can you also note that it corresponds to the rustdoc `-L` option (for a search term)?

Reply:
Sure!